#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_paths+files.dpatch by  <pdm@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use `speech-dispatcher' instead of `speechd' in paths.

@DPATCH@
Index: speech-dispatcher/config/speechd.conf.in
===================================================================
--- speech-dispatcher.orig/config/speechd.conf.in	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/config/speechd.conf.in	2010-06-22 19:20:26.000000000 +0200
@@ -67,7 +67,7 @@
 # The CustomLogFile allows logging all messages of the given kind,
 # regardless their priority, to the given destination.
 
-#CustomLogFile "protocol" "/var/log/speech-dispatcher/speechd-protocol.log"
+#CustomLogFile "protocol" "/var/log/speech-dispatcher/speech-dispatcher-protocol.log"
 
 # ----- VOICE PARAMETERS -----
 
@@ -93,7 +93,7 @@
 # default.  Voice types are symbolic names which map to particular
 # voices provided by the synthesizer according to the output module
 # configuratuion.  Please see the synthesizer-specific configuration
-# in etc/speechd/modules/ to see which voices are assigned to
+# in etc/speech-dispatcher/modules/ to see which voices are assigned to
 # different symbolic names.  The following symbolic names are
 # currently supported: MALE1, MALE2, MALE3, FEMALE1, FEMALE2, FEMALE3,
 # CHILD_MALE, CHILD_FEMALE
@@ -227,7 +227,7 @@
 #  - binary is the path to the binary executable of this module,
 #    either relative (to lib/speech-dispatcher-modules/) or absolute
 #  - configuration is the path to the config file of this module,
-#    either relative (to etc/speechd/modules/) or absolute
+#    either relative (to etc/speech-dispatcher/modules/) or absolute
 
 AddModule "espeak"       "sd_espeak"   "espeak.conf"
 AddModule "festival"     "sd_festival"  "festival.conf"
Index: speech-dispatcher/doc/speech-dispatcher.texi
===================================================================
--- speech-dispatcher.orig/doc/speech-dispatcher.texi	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/doc/speech-dispatcher.texi	2010-06-22 19:20:26.000000000 +0200
@@ -530,8 +530,8 @@
 
 @item
 Check the appropriate logfile in
-@file{~/.speech-dispatcher/logs/speechd.log} for user Speech
-Dispatcher or in @file{/var/log/speech-dispatcher/speechd.log}. Look
+@file{~/.speech-dispatcher/logs/speech-dispatcher.log} for user Speech
+Dispatcher or in @file{/var/log/speech-dispatcher/speech-dispatcher.log}. Look
 for lines containing the string 'ERROR' and their surrounding
 contents. If you hear no speech, restart Speech Dispatcher and look
 near the end of the log file -- before any attempts for synthesis of
Index: speech-dispatcher/src/server/options.c
===================================================================
--- speech-dispatcher.orig/src/server/options.c	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/src/server/options.c	2010-06-22 19:20:26.000000000 +0200
@@ -155,7 +155,7 @@
 	    exit(1);
 	  }
 	    
-	  debug_logfile_path = g_strdup_printf("%s/speechd.log",
+	  debug_logfile_path = g_strdup_printf("%s/speech-dispatcher.log",
 					       SpeechdOptions.debug_destination);
 	  /* Open logfile for writing */
 	  debug_logfile = fopen(debug_logfile_path, "wx");
Index: speech-dispatcher/src/server/set.c
===================================================================
--- speech-dispatcher.orig/src/server/set.c	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/src/server/set.c	2010-06-22 19:20:26.000000000 +0200
@@ -381,7 +381,7 @@
   if (!SpeechdOptions.debug && !debug) return 1;
  
   if (debug){
-    debug_logfile_path = g_strdup_printf("%s/speechd.log", SpeechdOptions.debug_destination);
+    debug_logfile_path = g_strdup_printf("%s/speech-dispatcher.log", SpeechdOptions.debug_destination);
 
     debug_logfile = fopen(debug_logfile_path, "w");
     if (debug_logfile == NULL){
Index: speech-dispatcher/src/server/speechd.c
===================================================================
--- speech-dispatcher.orig/src/server/speechd.c	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/src/server/speechd.c	2010-06-22 19:20:26.000000000 +0200
@@ -746,7 +746,7 @@
 void
 logging_init(void)
 {
-  char *file_name = g_strdup_printf("%s/speechd.log", SpeechdOptions.log_dir);    
+  char *file_name = g_strdup_printf("%s/speech-dispatcher.log", SpeechdOptions.log_dir);    
   assert(file_name != NULL);
   if (!strncmp(file_name,"stdout",6)){
     logfile = stdout;
Index: speech-dispatcher/src/python/speechd/paths.py
===================================================================
--- speech-dispatcher.orig/src/python/speechd/paths.py	2010-06-22 16:33:59.000000000 +0200
+++ speech-dispatcher/src/python/speechd/paths.py	2010-06-22 19:20:26.000000000 +0200
@@ -1 +1 @@
-SPD_SPAWN_CMD="/usr/local/bin/speech-dispatcher"
+SPD_SPAWN_CMD="/usr/bin/speech-dispatcher"
